multi-node/aws/pkg/config/config.go

```diff
@@ -108,6 +118,12 @@ type Cluster struct {
 	RecordSetTTL             int               `yaml:"recordSetTTL"`
 	HostedZone               string            `yaml:"hostedZone"`
 	StackTags                map[string]string `yaml:"stackTags"`
+	Subnets                  []Subnet          `yaml:"subnets"`
```

Comment:
I have added documentation in the commit https://github.com/coreos/coreos-kubernetes/pull/439/commits/7b5813187cf80101e5ad2def653f3b7f19357e84
Would you mind reviewing it? :bow: